### PR TITLE
[main] Add option to disable Octomap in Rviz Rendering Tools

### PR DESCRIPTION
**Pull request by AdamPettinger**
### Description
I was trying to use the `PlanningScene` Rviz display to show known collision objects, but NOT the octomap. To my knowledge, there was no way to have both. This PR adds an option to disable visualizing the octomap (see image)

image

### How to test
The easiest way I found to test is to use [Universal_Robots_ROS2_Driver](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver) and a [recorded bag file](https://drive.google.com/drive/folders/1iM7O1otpnuYmstDjhiwurQheFg8NczbV?usp=sharing) I took of the planning scene. That link should allow you to download the bag file

To set it up:
```sh
sudo apt update
sudo apt install ros-rolling-object-recognition-msgs ros-rolling-control-toolbox
cd $COLCON_WS/src #some workspace with moveit2 built
git clone https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver.git
vcs import --skip-existing --input Universal_Robots_ROS2_Driver/Universal_Robots_ROS2_Driver.repos
rosdep install -r --from-paths . --ignore-src --rosdistro $ROS_DISTRO -y
colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Now in 3 terminals:
```sh
ros2 launch ur_bringup ur_control.launch.py ur_type:=ur5e robot_ip:=yyy.yyy.yyy.yyy use_fake_hardware:=true launch_rviz:=false
```

```sh
ros2 launch ur_bringup ur_moveit.launch.py ur_type:=ur5e robot_ip:="xxx.xxx" use_fake_hardware:=true launch_rviz:=true
```

```sh
ros2 bag play octomap_viz_test/octomap_viz_test_0.db3
```

Under `MotionPlanning` -> `Scene Geometry` -> `Voxel Rendering`, try switching the type from the drop down menu. "Disabled" will ... disable the octomap

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/ros-planning/moveit/blob/master/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://ros-planning.github.io/moveit_tutorials/doc/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit_ros/visualization/rviz_plugin_render_tools/src/render_shapes.cpp [hunk 1]
```diff
@@ -156,11 +156,14 @@ void RenderShapes::renderShape(Ogre::SceneNode* node, const shapes::Shape* s, co
...
     case shapes::OCTREE:
     {
-      OcTreeRenderPtr octree(new OcTreeRender(static_cast<const shapes::OcTree*>(s)->octree, octree_voxel_rendering,
-                                              octree_color_mode, 0u, node));
-      octree->setPosition(position);
-      octree->setOrientation(orientation);
-      octree_voxel_grids_.push_back(octree);
+      if (octree_voxel_rendering != OCTOMAP_DISABLED)
+      {
+        OcTreeRenderPtr octree(new OcTreeRender(static_cast<const shapes::OcTree*>(s)->octree, octree_voxel_rendering,
```
Comment (henningkayser) [inline moveit_ros/visualization/rviz_plugin_render_tools/src/render_shapes.cpp:161]: Nit: use std::make_shared

File: moveit_ros/visualization/rviz_plugin_render_tools/src/render_shapes.cpp [hunk 2]
```diff
@@ -156,11 +156,14 @@ void RenderShapes::renderShape(Ogre::SceneNode* node, const shapes::Shape* s, co
 
     case shapes::OCTREE:
     {
-      OcTreeRenderPtr octree(new OcTreeRender(static_cast<const shapes::OcTree*>(s)->octree, octree_voxel_rendering,
-                                              octree_color_mode, 0u, node));
-      octree->setPosition(position);
-      octree->setOrientation(orientation);
-      octree_voxel_grids_.push_back(octree);
+      if (octree_voxel_rendering != OCTOMAP_DISABLED)
```
Comment (henningkayser) [inline moveit_ros/visualization/rviz_plugin_render_tools/src/render_shapes.cpp:159]: So.. when an already rendered OcTree is getting disabled, this won't take care of removing it, right?

Comment (AdamPettinger) [reply moveit_ros/visualization/rviz_plugin_render_tools/src/render_shapes.cpp:159]: I had thought so too, but switching to "Disabled" in RViz does in fact hide a previously rendered OcTree

Comment (henningkayser) [reply moveit_ros/visualization/rviz_plugin_render_tools/src/render_shapes.cpp:159]: nice!

**Review by JafarAbdi — APPROVED** [review]
LGTM

**Review by henningkayser — APPROVED** [review]

**Pull request merged**